### ISSUE TITLE
Missing git tag for releases

### ISSUE THREAD

**Issue by felixonmars** [pysimplesoap/pysimplesoap#166]
It would be nice to keep PyPI releases and git tags in sync :)

**Comment by sandrotosi** [comment]
yes, please tag accordingly on github when releasing something on PyPI - that would make packagers like a bit easier :)

also maybe it's time to cut a new release anyway? @reingart
